### PR TITLE
align node version between ci and prod

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

**Comment by AdaRoseCannon** [comment]
:+1: For merging

**Comment by AdaRoseCannon** [comment]
Fixes #66 
Fixes #64

File: tests/integration/lib/tabs.js [hunk 1]
```diff
@@ -42,5 +39,11 @@ module.exports = function(client) {
 		about() {
 			return loadTab('about');
 		}
+	};
+
+	module.exports = function () {
```
Comment (JakeChampion) [inline tests/integration/lib/tabs.js]: I don't think you can export a module from inside another module. What is meant to be happening here?

File: wdio.conf.js [hunk 2]
```diff
@@ -133,6 +143,10 @@ exports.config = {
 	// variables like `browser`. It is the perfect place to define custom commands.
 	before: function() {
 
+		const testWebsiteServer = express();
```
Comment (JakeChampion) [inline wdio.conf.js:148]: Nit picking: This looks like something `onPrepare` should handle

File: tests/integration/lib/tabs.js (hunk 1 above)
Comment (AdaRoseCannon) [inline tests/integration/lib/tabs.js]: It is overwriting the exports to take advantage of the singleton behaviour.

File: wdio.conf.js (hunk 2 above)
Comment (AdaRoseCannon) [inline wdio.conf.js:148]: Is onPrepare a webdriver or mocha thing?

Comment (JakeChampion) [inline wdio.conf.js:148]: Webdriver

File: tests/integration/lib/tabs.js [hunk 3]
```diff
@@ -2,45 +2,70 @@
...
+			return loaded;
 		}
 	}
+
+	const single = {
+		Tab,
+		tabs
+	};
+
+	module.exports = function () {
```
Comment (JakeChampion) [inline tests/integration/lib/tabs.js]: https://nodejs.org/api/modules.html#modules_caching

**Pull request merged**